_includes/header.html

```diff
@@ -1,8 +1,35 @@
 <header class="fx-row fx-row-between-xs fx-text-center">
-  <div class="fx-col fx-text-center"><a href="/{% if site.active_lang != site.default_lang %}{{ site.active_lang }}/{% endif %}">
-      <img alt="{{ site.data.general.nav.altJamulusIcon }}" width="40" height="40" src="{{ 'assets/img/jamulus-icon-2020.svg'| relative_url }}" /></a>
+  <div class="fx-col fx-text-center icon">
+    <a href="/{% if site.active_lang != site.default_lang %}{{ site.active_lang }}/{% endif %}">
+        <img alt="{{ site.data.general.nav.altJamulusIcon }}" width="40" height="40" src="{{ 'assets/img/jamulus-icon-2020.svg'| relative_url }}" />
+        <span>
+          <div>Jamulus</div>
+          <div class="tagline">{{ site.data.general.header.tagline }}</div>
+        </span>
+    </a> 
+      </div>
```

Comment:
Could you please check if the HTML is valid? It looks a bit suspicious to have this div indented here.

Reply:
I don't see why it wouldn't be valid - I think for markdown indents will break the code.

Anyways, I'll fix the indents to make it easier to read

Reply:
Great!